user/riff/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "riff"
+pkgver = "3.3.10"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable"]
+depends = ["rust-std"]
```

Comment:
i don't think this is depends

Reply:
sorry, this should be makedepends, right?